redisearch/query.py

```diff
@@ -288,6 +298,25 @@ def sort_by(self, field, asc=True):
         self._sortby = SortbyField(field, asc)
         return self
 
+    def set_params_dict(self, params: Dict[str, Union[str, int, float]]):
+        """"
+        Add a parameters dictionary. Overwrites an existing parameters dictionary.
+
+        - **params** - Dict[str, Union[str, int, float]] dictionary
+        """
+        self._params = params
+        return self
+
+    def add_param(self, param_name:str, value:Union[str, int, float]):
```

Comment:
It could also set a value for an existing param, no?
```suggestion
    def set_param(self, param_name:str, value:Union[str, int, float]):
```